.gitignore

```diff
@@ -24,7 +24,6 @@ jsons/
 report.html
 docs/_build
 webpack-stats.json
-dist
```

Comment:
I think this may have been an accident. It makes this PR massive with all the extra assets included from `dist/` in the PR.

Reply:
Removed from PR